src/types/redis_hash.h

```diff
@@ -56,7 +56,7 @@ class Hash : public SubKeyScanner {
   rocksdb::Status IncrByFloat(engine::Context &ctx, const Slice &user_key, const Slice &field, double increment,
                               double *new_value);
   rocksdb::Status MSet(engine::Context &ctx, const Slice &user_key, const std::vector<FieldValue> &field_values,
-                       bool nx, uint64_t *added_cnt);
+                       bool nx, uint64_t *added_cnt, uint64_t ttl = -1);
```

Comment:
```suggestion
                       bool nx, uint64_t *added_cnt, uint64_t expire = 0);
```
better to be 0 here.

Reply:
honestly I choose `-1` bc it turns out that in redis you can set negative ttl 🤷‍♂️ 
0 is fine with me.

Reply:
addressed via 76b25447b94878bfc75da31b30b6f7bcad5fce4f